USAGE.md

```diff
@@ -331,11 +331,12 @@ You can use the `aws-vault login` command to open a browser window and login to
 $ aws-vault login work
 ```
 
-If you have temporary STS credentials already available in your environment, you can have aws-vault use these credentials to sign you in.
-This is useful when you had to use something else than aws-vault to retrieve temporary credentials:
+If you have credentials already available in your environment, aws-vault will use these credentials to sign you in to the AWS console.
```

Comment:
Any way we can also support additional credentials source? (e.g. `--server` or `--ecs-server`)

Reply:
not at this time